### PR TITLE
London class _8-Marina_Almeida - JavaScript1-week_1

### PR DESCRIPTION
**Pull request by MarinaAlmeida20**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: mandatory/4-tax.js [hunk 1]
```diff
@@ -17,7 +20,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(n) {
+  return `\u00a3${Number(calculateSalesTax(n)).toFixed(2)}`;
```
Comment (Hatice06) [inline mandatory/4-tax.js:24]: Hi Marina, good job with using sterling currency code in your function.

File: mandatory/3-function-output.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ function combine2Words(word1, word2) {
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+  return firstWord.concat(" ", secondWord, " ", thirdWord);
```
Comment (jackhclee) [inline mandatory/3-function-output.js:14]: Please provide **comments** to these functions before these functions. Comments help programmer to understand the usage of the function and facilitate IDE auto suggestions / error detection.

File: mandatory/4-tax.js (hunk 1 above)
Comment (jackhclee) [inline mandatory/4-tax.js:24]: Great! You seems to know how to use Unicode escape right now!

File: mandatory/4-tax.js [hunk 3]
```diff
@@ -5,7 +5,10 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(n) {
+  let tax = n + 0.2 * n;
```
Comment (jackhclee) [inline mandatory/4-tax.js:9]: Think of a more concise expression here?

**Pull request closed without merging**